app/models/manageiq/providers/amazon/storage_manager/s3/cloud_object_store_container/operations.rb

```diff
@@ -0,0 +1,9 @@
+module ManageIQ::Providers::Amazon::CloudManager::CloudObjectStoreContainer::Operations
+  extend ActiveSupport::Concern
+
+  def raw_destroy
```

Comment:
`destroy` vs `delete` - it seems `raw_delete` is used more often than `raw_destroy`
`delete` would have the benefit of aligning more with `supports_delete`